lib/src/user.dart

```diff
@@ -0,0 +1,36 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2022 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+import 'native/realm_core.dart';
+
+/// This class represents a user in a MongoDB Realm app.
+/// A user can log in to the server and, if access is granted, it is possible to synchronize the local and the remote Realm.
```

Comment:
I know this is copied from the .NET docstring, but it is no longer correct - i.e. there are no remote Realms

```suggestion
/// A user can log in to the server and, if access is granted, it is possible to synchronize the local Realm to MongoDB.
```